build.gradle

```diff
@@ -175,8 +175,13 @@ subprojects {
             cacheChangingModulesFor 5, 'days'
         }
 
-        exclude(group: "ch.qos.logback", module: "logback-core")
-        exclude(group: "ch.qos.logback", module: "logback-classic")
+        if (!project.name.contains('cassandra-ticket-registry')) {
```

Comment:
Is this something we can work on and fix directly inside cassandra? Submit a patch to them to fix this logging dependency issue so we don't have to mess around with dependency gymnastics here? I realize you already have submitted a patch to them to take care of an issue, right? Could that also contain a fix for this? 

The alternative would be to remove the global exclusion rules for logback, and exclude them from every other module that wants to bring them in, run extensive tests to make sure everything works as before, and then move the cassandra-related changes into the cassandra module.